src/sql/workbench/contrib/dashboard/browser/dashboardRegistry.ts

```diff
@@ -35,7 +35,9 @@ export interface FlavorProperties {
 	databaseProperties: Array<Property>;
 	serverProperties: Array<Property>;
 	databasesListProperties?: Array<ObjectListViewProperty>;
+	databaseListHintText?: string;
```

Comment:
it would be nice to make the dashboard code a bit more generic (i.e. not so tied to database\server pages) at some point.

Reply:
yes, dashboard v3, we have identified some areas for improvement while working with Monica on the Kusto dashboard.